pages/interactions/application-commands.mdx

```diff
@@ -24,6 +24,7 @@ and can include spaces.
 | description         | string                                                                       | 1-100 character description for `CHAT_INPUT` commands, empty string for `USER` and `MESSAGE` commands | all         |
 | options?            | array of [application command option](#application-command-option-structure) | the parameters for the command, max 25                                                                | CHAT_INPUT  |
 | default_permission? | boolean (default `true`)                                                     | whether the command is enabled by default when the app is added to a guild                            | all         |
+| integration_types?  | array[[application integration type](#application-integration-types)]        | Contexts where the command can be used, if global; defaults `0`                                       | all         |
```

Comment:
```suggestion
| integration_types?  | array[[application integration type](#application-integration-types)]        | if the command is global, the contexts it can be used in (defaults to `0`)                                       | all         |
```